openml-lightgbm/src/main/java/com/feedzai/openml/provider/lightgbm/SWIGTrainResources.java

```diff
@@ -44,7 +44,7 @@
     /**
      * Handle for the output parameter necessary for the LightGBM dataset instantiation.
      */
-    public SWIGTYPE_p_p_void swigOutDatasetHandlePtr;
+    SWIGTYPE_p_p_void swigOutDatasetHandlePtr;
```

Comment:
Why do we need this to be public @AlbertoEAF ?

Reply:
The change is the opposite, we want it to be package-private.
Should we put it as `default`?

Reply:
We don't want the public, we're reverting (remove the 'public'). I don't know how to have it package-private and please codacy. Does that work? `default` as a keyword?

Reply:
João Sousa just explained me using this [link](https://docs.oracle.com/javase/tutorial/java/javaOO/accesscontrol.html).
There is no encapsulation for package-private. Therefore, we should ignore codacy, and let it cries as it wants.